Gopkg.lock

```diff
@@ -176,6 +177,22 @@
   pruneopts = "NUT"
   revision = "3165313d6d3f973ec0b0ed3ec5a63b520e065d40"
 
+[[projects]]
+  digest = "1:63ede27834b468648817fb80cfb95d40abfc61341f89cb7a0d6779b6aa955425"
+  name = "github.com/google/go-github"
```

Comment:
Any idea what is pulling in this dependency? It seems unusual to include a github client into minikube.

Reply:
it looked weird to me as well, and I honestly don't know where it's coming from. 

Before submitting the PR I double checked and it is not coming from our `github.com/machine-drivers/docker-machine-driver-vmware` import, so I assumed that the `dep ensure` command pulled a dependency coming from somewhere else in the minikube codebase.